pkg/apis/scorecard/v1v2conversions.go

```diff
@@ -17,17 +17,11 @@ package scorecard
 import (
 	scapiv1alpha1 "github.com/operator-framework/operator-sdk/pkg/apis/scorecard/v1alpha1"
 	scapiv1alpha2 "github.com/operator-framework/operator-sdk/pkg/apis/scorecard/v1alpha2"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func ConvertScorecardOutputV1ToV2(v1ScorecardOutput scapiv1alpha1.ScorecardOutput) scapiv1alpha2.ScorecardOutput {
 
-	output := scapiv1alpha2.ScorecardOutput{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "ScorecardOutput",
-			APIVersion: "osdk.openshift.io/v1alpha2",
-		},
-	}
```

Comment:
Just wanted to point out that the rationale for not initializing TypeMeta for a struct because you expect the controller-runtime cache/client to initialize it doesn't hold here.

In this case the `scapiv1alpha1.ScorecardOutput{}` struct is only being marshalled to JSON or text and not being sent to the server. I'll let @jmccormick2001 chime if that's not the case.
https://github.com/operator-framework/operator-sdk/blob/e03cde67f916af43b9e15edf6b976ec9894b23e6/internal/scorecard/scorecard_output.go#L37-L68

But in any case removing it doesn't seem to be an issue so it seems fine to me for this case.

Reply:
@hasbro17 @camilamacedo86 Actually I think this is an issue, because we can output scorecard results as `v1alpha1` or `v1alpha2` and the type meta is what needs to be there to distinguish it.

@camilamacedo86 Can you make another PR that re-adds the type meta for the scorecard types?

Reply:
Done here: https://github.com/operator-framework/operator-sdk/pull/2159 and make clear that it is mandatory and why.